<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit</class>
 <widget class="QDialog" name="edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>edit_table</string>
  </property>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>670</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>70</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add new table</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_view">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>1161</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EDIT TABLES</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
